ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -102,29 +104,33 @@ TFCore::loadModel ()
   Status status;
   GraphDef graph_def;
 
+  if (!g_file_test (model_path, G_FILE_TEST_IS_REGULAR)) {
+    g_message ("the file of model_path is not valid\n");
+    return -1;
+  }
   status = ReadBinaryProto (Env::Default (), model_path, &graph_def);
   if (!status.ok()) {
-    GST_ERROR ("Failed to read graph.\n%s", status.ToString().c_str());
-    return -1;
+    g_message ("Failed to read graph.\n%s", status.ToString().c_str());
```

Comment:
I have one question. As we all know, we have used "GST_DEBUG=**" to do a debugging  in case of the error case while running nnstreamer facility. What is difference between GST_ERROR and g_message(3)? 

* https://github.com/nnsuite/nnstreamer/blob/master/tools/debugging/README.md#displaying-debug-messages-with-gst_debug

Reply:
g_message is for logging the normal messages. IMO, g_critical is more likely to fit here.

Reply:
@leemgs 
If we take `GST_DEBUG`, or `GST_ERROR`, to get logs it requires an option.
However, `g_message` prints log without any options.
In this case, in my opinion, because it is a critical, generating logs without any option looks proper.

@wooksong 
It looks better. I'll apply it.